src/main/paradox/running_locally/docker/docker-compose-32GB.yaml

```diff
@@ -0,0 +1,108 @@
+version: "3.3"
+services:
+  kg:
+    image: "bluebrain/kg-nexus:latest"
+    deploy:
+      resources:
+        limits:
+          memory: 6G
+    environment:
+      - "CASSANDRA_CONTACT_POINT1=cassandra:9042"
+      - "BIND_INTERFACE=0.0.0.0"
+      - "PUBLIC_URI=http://localhost"
+      - "IAM_BASEURI=http://iam:8080/v0"
+      - "SPARQL_BASE_URI=http://blazegraph:9999/blazegraph"
+      - "SPARQL_ENDPOINT=http://blazegraph:9999/blazegraph/sparql"
+      - "ELASTIC_BASE_URI=http://elasticsearch:9200"
+      - "JAVA_OPTS=-Xms2G -Xmx4G"
+
+  iam:
+    image: "bluebrain/iam-nexus:latest"
+    deploy:
+      resources:
+        limits:
```

Comment:
could we remove the limits across the entire file to avoid cases where the process gets killed because of off-heap memory usage? limiting the heap should suffice